src/fragments/lib/analytics/flutter/identifyuser.mdx

```diff
@@ -1,27 +1,37 @@
 This call sends information that you have specified about a user to Amazon Pinpoint. This could be for an unauthenticated (guest) or an authenticated user.
 
-You can get the current user's ID from the Amplify Auth category as shown below. Be sure you have it added and setup per the Auth category documentation.
+You can get the current user's ID from the Amplify Auth category as shown per the Auth category documentation. Be sure to have it before you set it below.
```

Comment:
Nit: be sure to have it setup before you use it as shown below. (And then we can link to getting started with Auth). @salihgueler